server/gql/schema.js

```diff
@@ -124,13 +126,24 @@ const upsertResolver = async ({
         requestMapping,
         responseMapping,
         type,
-        field
+        field,
+        publish: publisher
     };
 
     let result;
     if (id) {
         const updated = await resolver.findById(id);
         result = updated.update(properties);
+        if (properties.publish) {
+            const subExists = await subscription.findOne({ where: { field: properties.publish } });
```

Comment:
So this is actually a pretty cool candidate for the `findOrCreate`. You can see this in the sequelize docs here: http://docs.sequelizejs.com/manual/tutorial/models-usage.html

Not necessary though